### PR TITLE
Liked page

### PR DESCRIPTION
**Pull request by agrawalshanu708**
Create Like page,
User can save  favourite items also remove from there

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,14 +3,15 @@ import logo from "./logo.png";
 import { Routes, Route, Link } from "react-router-dom";
 import {Navbar} from "./components/index"
 import {Searchbar} from "./components/index"
```
Comment (zabihshaik) [inline src/App.js:5]: Can have Navbar, Searchbar in 1 line (import) from index

File: src/pages/likedpage/likecard.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import {AiOutlineHeart} from "react-icons/ai"
+import {MdOutlinePlaylistPlay} from "react-icons/md"
+import {MdOutlineWatchLater} from"react-icons/md"
```
Comment (zabihshaik) [inline src/pages/likedpage/likecard.jsx:4]: Can have line 3, line 4 in 1 import

**Review by zabihshaik — APPROVED** [review]
Everything looks good. 
UI is good.
Just a few suggestions.

**Review by harshitpaliwal95 — APPROVED** [review]
good to go
ui looks clean

**Pull request merged**